src/http/StorageConfigEndpoints.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+
+const createHandler = (storageConfig) => {
+    return (req, res) => {
+        const { id, partition } = req.params
+        const body = {
+            persistent: storageConfig.hasStream({
+                id,
+                partition
+            })
+        }
+        res.status(200).send(body)
+    }
+}
+
+module.exports = (storageConfig) => {
+    const router = express.Router()
+    const handler = (storageConfig !== null) ? createHandler(storageConfig) : (_, res) => res.status(501).send('Not a storage node.')
+    router.get('/streams/:id/storage/partitions/:partition', handler)
```

Comment:
I'm personally a fan of using good-old query parameters when we are not specifically using URLs to represent resources.

E.g.
```
/checkIsStoring?id=7wa7APtlTq6EC5iTCBy6dw&partition=0
```